crates/sui-benchmark/tests/simtest.rs

```diff
@@ -563,32 +564,56 @@ mod test {
         .await
         .unwrap();
 
-        let driver = BenchDriver::new(5, false);
+        let bench_task = tokio::spawn(async move {
+            let driver = BenchDriver::new(5, false);
 
-        // Use 0 for unbounded
-        let test_duration_secs = get_var("SIM_STRESS_TEST_DURATION_SECS", test_duration_secs);
-        let test_duration = if test_duration_secs == 0 {
-            Duration::MAX
-        } else {
-            Duration::from_secs(test_duration_secs)
-        };
-        let interval = Interval::Time(test_duration);
-
-        let show_progress = interval.is_unbounded();
-        let (benchmark_stats, _) = driver
-            .run(
-                vec![proxy],
-                workloads,
-                system_state_observer,
-                &registry,
-                show_progress,
-                interval,
+            // Use 0 for unbounded
+            let test_duration_secs = get_var("SIM_STRESS_TEST_DURATION_SECS", test_duration_secs);
+            let test_duration = if test_duration_secs == 0 {
+                Duration::MAX
+            } else {
+                Duration::from_secs(test_duration_secs)
+            };
+            let interval = Interval::Time(test_duration);
+
+            let show_progress = interval.is_unbounded();
+            let (benchmark_stats, _) = driver
+                .run(
+                    vec![proxy],
+                    workloads,
+                    system_state_observer,
+                    &registry,
+                    show_progress,
+                    interval,
+                )
+                .await
+                .unwrap();
+
+            // TODO: make this stricter (== 0) when we have reliable error retrying on the client.
+            tracing::info!("end of test {:?}", benchmark_stats);
+            assert!(benchmark_stats.num_error_txes < 100);
+        });
+
+        let surfer_task = tokio::spawn(async move {
+            // now do a sui-surfer test
+            let mut path = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+            path.extend([
```

Comment:
should we include here all the projects under tests?
I added the tests for Random as another Move project, to be able to isolate its testing

Reply:
Done